src/iuse.cpp

```diff
@@ -6635,20 +6635,35 @@ int iuse::spray_can(player *p, item *it, bool, const tripoint& )
         }
     }
 
-    std::string message = string_input_popup(ismarker ? _("Write what?") : _("Spray what?"),
-                          0, "", "", "graffiti");
+    return handle_ground_graffiti(p, it, ismarker ? _("Write what?") : _("Spray what?"));
+}
+
+int iuse::handle_ground_graffiti(player *p, item *it, const std::string prefix)
+{
+    const auto suffix = pgettext("mind the starting space", " (To delete, input one '.')");
+    std::string message = string_input_popup( prefix + suffix, 0, "", "", "graffiti" );
```

Comment:
If the space is so important and is required, why not add it in the code: `( prefix + " " + suffix, `? Trailing/leading white spaces are considered harmful (#4653) by the translators.